modules/hal_nxp/mcux/mcux-sdk-ng/drivers/drivers.cmake

```diff
@@ -186,6 +186,12 @@ endif()
   set_variable_ifdef(CONFIG_HAS_MCUX_XCACHE		CONFIG_MCUX_COMPONENT_driver.cache_xcache)
 
 if((${MCUX_DEVICE} MATCHES "MIMX9596") OR (${MCUX_DEVICE} MATCHES "MIMX8UD7") OR (${MCUX_DEVICE} MATCHES "MIMXRT118") OR (CONFIG_SOC_MIMX94398) OR (CONFIG_SOC_MIMXRT798S))
+
+  if(CONFIG_SOC_MIMX94398_M33)
+    set(CONFIG_MCUX_COMPONENT_driver.irqsteer_1 ON)
```

Comment:
Why is this change needed if you do not have a dependency on the new irq_steer driver?

Reply:
> Why is this change needed if you do not have a dependency on the new irq_steer driver?
This is a irqsteer driver(CONFIG_MCUX_COMPONENT_driver.irqsteer_1) in mcux sdk. It needs to be enabled by default.
The i2c driver(drivers/i2c/i2c_mcux_lpi2c.c) of zephyr will use lpi2c driver(drivers/lpi2c/fsl_lpi2c.c) of mcux sdk,
the lpi2c driver of mcux sdk will can api EnableIRQ(drivers/common/fsl_common_arm.h) of mcux sdk,
EnableIRQ -> IRQSTEER_EnableInterrupt
<img width="2486" height="731" alt="image" src="https://github.com/user-attachments/assets/fef82d1c-0f59-4766-bda8-32d21e0cd079" />